### PR TITLE
[fixed] Fix windows file naming errors

### PR DESCRIPTION
**Pull request by enomothem**
Fix windows file naming errors https://github.com/pwnwriter/haylxon/issues/36
在Windows中，以下字符是特殊命名的，不能用于文件名或目录名：
```
<（小于号）
>（大于号）
:（冒号）
"（双引号）
/（正斜杠）
\（反斜杠）
|（管道符号）
?（问号）
*（星号）
```

### CODE REVIEW

**Comment by pwnwriter** [comment]
I'll have to review on this. I'm currently out , I'll check back and will notify you very soon, And thank you for the pr.

File: src/cli/screenshot.rs [hunk 1]
```diff
@@ -143,11 +144,14 @@ async fn take_screenshot(
         .http1_ignore_invalid_headers_in_responses(true)
         .trust_dns(true)
         .build()?;
+    let re = Regex::new(r"[<>?.~!@#$%^&*\/|;:']").unwrap();
```
Comment (pwnwriter) [inline src/cli/screenshot.rs]: I'm not sure, if it's good to use a new crate as a dependency for this only. Do you have any other implementations @enomothem ?

Comment (enomothem) [reply src/cli/screenshot.rs]: @pwnwriter  If you don't use regular, you'll need to use replace() more than once

Comment (pwnwriter) [reply src/cli/screenshot.rs]: Btw, I'm afraid I don't have a Windows machine to test this pr. Give me some time.

**Review by pwnwriter — APPROVED** [review]

**Pull request merged**